tests/bluetooth/tester/src/audio/btp_bap_broadcast.h

```diff
@@ -41,6 +42,9 @@ struct btp_bap_broadcast_remote_source {
 };
 
 struct btp_bap_broadcast_local_source {
+	bool allocated;
+	uint8_t source_id;
```

Comment:
`source_id` is still sometimes used as an ID (btp_bap_broadcast_local_source_from_src_id_get), and sometimes used as an index (cap_broadcast_params[source->source_id]). 

It should be either or, not both :)

Reply:
Just FYI, we also have macros like `ARRAY_INDEX` that can return the index of an element in a struct, so it's only useful to store the index if we for some reason have neither index or a pointer to a `btp_bap_broadcast_local_source`

Reply:
Didn't know about the ARRAY_INDEX, that was a good tip. Then I'll use something like:
```C
uint8_t idx = ARRAY_INDEX(local_sources, source);
something cap_broadcast_params[idx] something_more
```

Reply:
Shouldn't be anymore uses of source_id as index anymore now.